services/frontend-service/src/ui/components/ReleaseDialog/ReleaseDialog.scss

```diff
@@ -1,29 +1,47 @@
 @use '@material/typography/mdc-typography';
 @use '@material/card/mdc-card';
 
-.release-dialog-header {
-    margin: 10px;
+.release-dialog-commitId {
+    border: 1px solid white;
+    border-radius: 15px;
+}
+
+.release-dialog {
+    border-radius: 25px 25px 0px 0px;
 }
 
+.release-dialog div div {
+    border-radius: 25px 25px 0px 0px;
+}
```

Comment:
this seems unnecessary, as `.release-dialog` has the same values.
Does it work when you remove this?

Reply:
The release dialog renders a div inside it which is shown as a white background, U had to add this otherwise there will be a white background behind the header